netflix/src/components/App.tsx

```diff
@@ -1,33 +1,30 @@
-import React, { useState, useCallback } from "react";
+import React from "react";
 import TopBar from "./top/TopBar";
 import VideoBar from "./main/VideoBar";
 import { ErrorBoundary } from "react-error-boundary";
 import ErrorFallback from "./err/ErrorFallback";
 import EditMovie from "./filmInteraction/edit/EditMovie";
 import DeleteMovie from "./filmInteraction/delete/DeleteMovie";
+import AddMovie from "./filmInteraction/add/AddMovie";
+import Details from "./details/Details";
+import { useSelector } from "react-redux";
 
 const App: React.FC = () => {
-  const [edit, setEdit] = useState(false);
-  const [deleted, setDeleted] = useState(false);
+  const { deleteMovie, editMovie, addMovie, blur } = useSelector(
```

Comment:
Better to always create selectors outside of components and reuse them across application. Otherwise you will face code duplication and unnecessary rendering. There is even specific library `reselect` with main purpose to fix above issues.